plugins/spu2-x/src/Windows/SndOut_waveOut.cpp

```diff
@@ -239,14 +239,12 @@ class WaveOutModule : public SndOutModule
                 wmId = LOWORD(wParam);
                 wmEvent = HIWORD(wParam);
                 switch (wmId) {
-                    //case TB_ENDTRACK:
-                    //case TB_THUMBPOSITION:
                     case TB_LINEUP:
                     case TB_LINEDOWN:
                     case TB_PAGEUP:
                     case TB_PAGEDOWN:
                         wmEvent = (int)SendMessage((HWND)lParam, TBM_GETPOS, 0, 0);
-                    case TB_THUMBTRACK:
+                    case TB_THUMBPOSITION:
```

Comment:
My suggestion was that you have both TB_THUMBTRACK and TB_THUMBPOSITION here, not just TB_THUMBPOSITION.

You should probably add TB_TOP and TB_BOTTOM above the TBM_GETPOS line too..

Reply:
OK, I'll do that.